js/form_handler.js

```diff
@@ -93,11 +93,23 @@ $(function()
 			return this.options.field_set.triggerHandler('get_current_values');
 		},
 
+		_updatePreviousValues: function()
+		{
+			let me = this;
+			if(this.element.find('[data-attribute-previous-value]').length>0) {
+				let aPreviousValues = {};
+				$(this.element.find('[data-attribute-previous-value]')).each(function (idx, elt) {
+					aPreviousValues[$(elt).data('field-id')] = $(elt).data('attribute-previous-value');
+				});
+				me.element.find('[data-field-id=previous_values]').find('input[type=hidden]').val(JSON.stringify(aPreviousValues));
```

Comment:
Please put quotes in selectors, we discussed about this several times.

```suggestion
				me.element.find('[data-field-id="previous_values"]').find('input[type="hidden"]').val(JSON.stringify(aPreviousValues));
```